src/components/breadcrumbs/_variables.scss

```diff
@@ -12,10 +12,11 @@
 $ouiBreadcrumbSpacing: $ouiSizeS !default;
 $ouiBreadcrumbTruncateWidth: $ouiSize * 10 !default;
 
-$ouiBreadcrumbBlueBackground: #B9D9EB !default;
-$ouiBreadcrumbGrayBackground: #D9E1E2 !default;
-$ouiBreadcrumbCollapsedLink: #002A3A !default;
-$ouiBreadCrumbHoverColor: #535861 !default;
+$ouiBreadcrumbBlueBackground: tintOrShade($ouiColorPrimary, 72%, 41%) !default;
+$ouiBreadcrumbGrayBackground: tint($ouiColorLightShade, 14%) !default;
+$ouiBreadcrumbCollapsedLink: shadeOrTint($ouiColorPrimaryText, 61%, 20%) !default;
+$ouiBreadcrumbTextColor: makeHighContrastColor($ouiTextSubduedColor, $ouiBreadcrumbGrayBackground) !default;
```

Comment:
Along with the naming suggested above, this would become `$ouiBreadcrumbInactiveTextColor`, because we don't use it for the last one, right?

Reply:
Correct, will be fixed in next commit